### PR TITLE
9.0-mig-report_custom_filename

### PR DESCRIPTION
**Pull request by susport**
Change to version 9.0

### CODE REVIEW

**Comment by rvalyi** [comment]
:+1:

**Comment by pedrobaeza** [comment]
There's another opened PR for this module migration in https://github.com/OCA/reporting-engine/pull/52. Have you reviewed it?

**Comment by susport** [comment]
Closed because existing PR here [(https://github.com/OCA/reporting-engine/pull/52)]

**Comment by pedrobaeza** [comment]
I reopen this PR as there's no answer from original author.

File: report_custom_filename/controllers/report_controller.py [hunk 1]
```diff
@@ -19,7 +19,7 @@
 #
 ##############################################################################
```
Comment (lasley) [inline report_custom_filename/controllers/report_controller.py]: Please change all headers to the [new shorter version](https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3)

**Review by lasley — APPROVED** [review]
Thanks @susport - code 👍  outside of the one comment

**Review by nicomacr — APPROVED** [review]

**Review by mdietrichc2c — APPROVED** [review]
Apart from @lasley 's comment on headers, :+1: from me (also, I tested the module on a local instance).

**Pull request merged**